### DISCUSSION TITLE
Add a toggle in the settings to stop previewing mods in the beatmap carousel

### DISCUSSION THREAD

**Discussion by Bruno5430** [ppy/osu#24103] [Ideas]
Lazer always previews mods in the beatmap carousel, mods such as DT, NC, HT, DC, WU, WD, and MU(ted) changes the pitch, speed or volume of the songs. Some people like it, some doesn't, and some get irritated over time.
This behavior should be toggled by the user's preference and not always active.
Also, if in the future, gameplay can be previewed in the carousel directly and mods are also previewed (example: HDHR) it will also be nice for it to be toggled in the settings.

**Comment by nagi-desuuu** [comment]
But I thought that people already collectively decided to disagree to that?
Look, the point of mod preview is to hear if the song's gonna be fast or not **before** playing the map; if anything, I'd say that this was deliberate.

**Comment by peppy** [comment]
Can you better explain this?

Why does the song becoming "annoying" at song select but not when you play? I can't really understand.

**Reply by cihe13375** [reply]
I'm personally neutral on this purposal, but please note that players can (sometimes unconsciously) "click through" many maps before finally choose one to play, and clicking on a map automatically begins previewing that song, while not all music are good with... e.g. NC.

**Reply by peppy** [reply]
> I'm personally neutral on this purposal, but please note that players can (sometimes unconsciously) "click through" many maps before finally choose one to play, and clicking on a map automatically begins previewing that song, while not all music are good with... e.g. NC.

makes a very good argument for keeping things how it is - the user could preview the sound without playing the each beatmap to find one that "sounds good".
